lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
> statically include them and brfs inlines them

Yep. But we can't ignore them. Remember, we still need those scripts for the "export as html" option. When users click that, we create a full version of the report as if it was generated by the CLI/extension. So it needs all the same juice. It's the [setup work](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/report/templates/report-template.html#L228-L247) that we leave out.

One day I will draw you a 🖼 ...on 🗒 .